src/main/java/jmgomez/apipolicy/controller/PolicyController.java

```diff
@@ -20,18 +20,18 @@ public PolicyDto[] getPolicies() {
     }
 
     @GetMapping("/policies/{id}")
-    public PolicyDtoCov getPolicyByIDs(@PathVariable String id) {
-        return policyService.getPoliciesByIDs(getUserId());
+    public PolicyDtoCov getPolicyByIDs(@PathVariable("id") String id) {
+        return policyService.getPoliciesByIDs(getUserId() + id);
```

Comment:
????  Why? I think you are coupling your logic to the mock's logic. You get an ID, you use that ID. Full stop.